.github/workflows/test.yaml

```diff
@@ -45,19 +45,19 @@ jobs:
         uses: actions/setup-python@v2
```

Comment:
Can we please upgrade to the latest version of the action, think it is v4?

Reply:
Done.